src/Umbraco.Tests/Testing/TestOptionAttributeBase.cs

```diff
@@ -28,9 +31,17 @@ public static TOptions GetTestOptions<TOptions>()
             var test = TestContext.CurrentContext.Test;
             var typeName = test.ClassName;
             var methodName = test.MethodName;
-            var type = Type.GetType(typeName, true);
+            var type = Type.GetType(typeName, false);
             if (type == null)
-                throw new PanicException($"Could not resolve the type from type name {typeName}"); // makes no sense
+            {
+                type = ScanAssemblies
```

Comment:
It would be better if you change the List to be a list of `Assembly`, then you would use the `Assembly.GetType` method which means the type didn't have to be pre-loaded in the AppDomain. The problem with `Type.GetType` is that it will ONLY find types that have already been loaded in the AppDomain, it will not find other ones until they are. This is why the TypeFinder and MVC's own assembly scanning uses the BuildManager which will look through every assembly and it's types.